manager/controllers/app/blueprint_controller.go

```diff
@@ -376,7 +375,11 @@ func (r *BlueprintReconciler) reconcile(ctx context.Context, cfg *action.Configu
 	// the status is unknown yet - continue polling
 	if blueprint.Status.ObservedState.Error == "" {
 		log.Trace().Msg("blueprint.Status.ObservedState is not ready, will try again")
-		return ctrl.Result{RequeueAfter: 2 * time.Second}, nil
+		interval, err := environment.GetResourcesPollingInterval()
+		if err != nil {
+			log.Error().Err(err).Msg("error getting resource polling interval :" + err.Error())
```

Comment:
The error here is meaningless since the operation will go through. It is better to print the error when there is a printout of all env vars used by manager. Something in light of "error getting the value. Setting the default to ..."